### DISCUSSION TITLE
Documents are blank on MacOS

### DISCUSSION THREAD

**Discussion by havedill** [docmost/docmost#362] [Help / Questions]
I can modify documents on my Windows 11 PC in chrome, and in my android chrome browser i can see the documents. However, on MacOS, the documents are all blank.

I'm running a docker container of docmost with a docker postgresql 16. I'm also reverse proxied through nginx proxy manager if that matters, for https purposes.

**Comment by havedill** [comment]
OK, it definitely seems related to the reverse proxy. If i take that out of the picture i can see the documents updating across platforms.

It must be caching or something? Not sure if there is a code fix docmost can make there

**Comment by Philipinho** [comment]
Make sure websocket is enabled on your NGINX proxy.
